client/src/components/common/Filter/index.js

```diff
@@ -1,70 +1,93 @@
 import React, { Component } from 'react';
+
+import axios from 'axios';
 import PropTypes from 'prop-types';
 
 import { Select, Form, Button, Switch } from 'antd';
-import { cities, types, ranges } from './staticData';
+import Loader from '../Loader';
+import { types, ranges } from './staticData';
 import './style.css';
 
 const { Option } = Select;
 
 class Filter extends Component {
+  state = {
+    cities: [],
+  };
+
+  componentDidMount = async () => {
+    const result = await axios.get('/api/v1/cities');
+    this.setState(prevState => ({
```

Comment:
If there is no need to previous state use this way please
```suggestion
    this.setState({ cities: [...result] });
```